package.json

```diff
@@ -67,12 +67,11 @@
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
     "rollup": "^4.32.0",
-    "rollup-plugin-babel": "^4.4.0",
```

Comment:
this plugin is still in use and referenced in https://github.com/ianmiller347/feather-icons-react/blob/master/rollup.config.mjs

Reply:
ah nevermind. i got it confused with https://www.npmjs.com/package/@rollup/plugin-babel